addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -441,12 +446,17 @@ export const editorCommands = {
         });
         editor.document.execCommand('removeFormat');
```

Comment:
Actually it would probably be clearer to add the `historyPauseSteps` and its comment right before this line. We won't need it before and grouping them makes it easier to understand. Could you please do that and test that it does indeed change nothing?

Reply:
@Zynton Changes done, please have a look.

Reply:
Did you test that it still worked the same?

Reply:
@Zynton 
Yes, I tested it works the same. 
[Video](https://drive.google.com/file/d/1VLTNc4sMaJr3KQ8JbkzOuSARubH1gdib/view)

Also, if I place the `historyUnpause()` after the `document.execCommand`, it works the same. What should be the convention here? Should I move it up, or keep it at the end? In my opinion, there will be no difference in both.

Reply:
You can keep it there atm.